README.md

```diff
@@ -86,6 +86,7 @@ export function effect(callback) {
   return () => {
     w.unwatch(computed);
     typeof cleanup === "function" && cleanup();
+    cleanup = undefined;
```

Comment:
Since `cleanup` can be any arbitrary user-land function reference, and since the returned function here is being sent out to user-land, there's an opportunity for users to accidentally capture resources they meant to clean up in a closure... albeit very unlikely, this one line is a reasonable defense.